requirements/extras/uvloop.txt

```diff
@@ -0,0 +1 @@
+uvloop==0.15.3
```

Comment:
We might want to loosen the versioning on this (`uvloop>=0.15.3`) - or perhaps removing it all together so it's just `uvloop`.

Otherwise it could cause issues when installing it with other libraries which are dependent on uvloop.

I imagine the uvloop API will stay pretty stable, as far as we need it, so should be safe enough.

Reply:
Thanks. That's a good point. I did a quick search and it seems `uvloop.install()` was introduced in 0.12.0 according to [this](https://github.com/MagicStack/uvloop/commit/a3d8d4011fe432d7c83b80ecac8e678257b384f0#diff-62ea7a112b3168cba4028fb6d7d3f8763d426e4fbb96bca64396541b6be1af92)